authenticate-oauth/Web/Authenticate/OAuth.hs

```diff
@@ -103,6 +103,8 @@ data OAuth = OAuth { oauthServerName      :: String -- ^ Service name (default:
                    -- ^ Optional authorization realm (default: @Nothing@)
                    , oauthVersion         :: OAuthVersion
                    -- ^ OAuth spec version (default: 'OAuth10a')
+                   , oauthBodyHashMethod  :: BodyHashMethod
```

Comment:
Since the constructor isn't exposed, this is a feature addition - so a minor version bump